packages/fluent-theme/src/styles/index.ts

```diff
@@ -0,0 +1,11 @@
+import { useMemo } from 'react';
+import useStlyeToEmotionObject from 'botframework-webchat-component/lib/hooks/internal/useStyleToEmotionObject';
```

Comment:
```suggestion
import useStyleToEmotionObject from 'botframework-webchat-component/lib/hooks/internal/useStyleToEmotionObject';
```

We need to think about how to expose Emotion.

It got nonce... which means, the web devs blessed Web Chat to create CSS, but not random people.

Reply:
Please let me know if you have a better idea on this

Reply:
Naming fixed